tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
   "include": ["**/*"],
+  "exclude": ["node_modules/**/*", "dist/**/*"],
   "compilerOptions": {
     "alwaysStrict": true,
     "declaration": false,
     "forceConsistentCasingInFileNames": true,
     "noImplicitAny": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "skipLibCheck": false,
+    "skipLibCheck": true,
```

Comment:
🤔  why do we need to skip lib check here?

Reply:
Just saw under risks, nvm

Reply:
Left comment in the PR description 😏

Reply:
We might be able to fix it with a 3rd party type library, maybe `@types/svg-sprite`, but this is fine for now

Reply:
We use it already, but it is failing... That is, the types themselves do not pass the check.